addon/models/preprint.js

```diff
@@ -36,6 +36,16 @@ export default OsfModel.extend(ContributorMixin, {
     description: DS.attr('fixstring'),
     tags: DS.attr(),
     public: DS.attr('boolean'),
+    /* Sloan fields */
+    hasCoi: DS.attr('boolean'),
+    hasDataLinks: DS.attr('boolean'),
+    hasPreregLinks: DS.attr('boolean'),
+    dataLinks: DS.attr(),
```

Comment:
Are these returned as an array from the API? Is there an `array` type attr?

Reply:
Yes. There is [ember-data-array-transform](https://emberobserver.com/addons/ember-data-array-transform), but it [doesn't do much](https://github.com/PrecisionNutrition/ember-data-array-transform/blob/master/addon/transforms/array.js).